dockerfiles/idp_prod.Dockerfile

```diff
@@ -28,15 +96,14 @@ ENV ASSET_HOST http://localhost:3000
 ENV DOMAIN_NAME localhost:3000
 ENV PIV_CAC_SERVICE_URL https://localhost:8443/
 ENV PIV_CAC_VERIFY_TOKEN_URL https://localhost:8443/
+ENV REMOTE_ADDRESS_HEADER X-Forwarded-For
 
 # Install dependencies
 RUN apt-get update && \
     apt-get install -y \
     git-core \
-    git-lfs \
     curl \
     zlib1g-dev \
-    build-essential \
```

Comment:
I think we may be able to remove `libsqlite3-dev` and `sqlite3` as well?

Reply:
I'll give that a try!  Though I do need to spend some time to figure out why my tstest cluster is sad so that I can test it first.

Reply:
Seemed to work!  I was able to log in and add/delete different auth methods without it throwing any errors at least.